### ISSUE TITLE
[720pier] Login failed: Selector ":has(a[href^="./ucp.php?mode=logout&"])" didn't match

### ISSUE THREAD

**Issue by whitesnakeftw** [Jackett/Jackett#11828]
### Environment

**OS**: Windows 10.0.19041.985

**.Net Runtime**: [.Net-Core/.Net-Framework/Mono]

**.Net Version**:

**Jackett Version**: v0.18.181

**Last Working Jackett Version**:

**Are you using a proxy or VPN?** [yes/no] no

### Description

Can't login anymore. I noticed that the usual cookie pattern has changed: it doesn't start with the `phpbb3_h6m7i_u` value anymore, but with `_ym_uid` and_`ym_d` and thought this may be related.

### Logged Error Messages

`Login failed: Selector ":has(a[href^="./ucp.php?mode=logout&"])" didn't match`

**Comment by ilike2burnthing** [comment]
In browser (making sure you're definitely on https://720pier.ru/) sign out and sign back in again, see if that affects the cookie. If not, try signing in anew from a private/incognito window.

I have no issues with the indexer in Jackett, and the site's cookies are using the `phpbb3_h6m7i_*` format for me.

**Comment by whitesnakeftw** [comment]
@ilike2burnthing I managed to fix it eventually. I had already tried logging out and in again, but it didn't help. I also tried to log in in incognito mode and that changed things, meaning that I wasn't getting the long version of the cookie with the `_ym_` parameters anymore, but even this new short version of the cookie wasn't working.

I then tried to manually delete all the cookies from my browser, but it still didn't work. And I noticed that my `phpbb3_h6m7i_k` value was empty. Then I randomly noticed that at the bottom of the site there's a "Delete cookies" button, so I hit that and after logging back in I finally got the full and working version of the cookie. Sorry for the hassle.
